docs/flowcollector-flows-netobserv-io-v1alpha1.adoc

```diff
@@ -0,0 +1,1327 @@
+// Automatically generated by 'openshift-apidocs-gen'. Do not edit.
+:_content-type: ASSEMBLY
+[id="flowcollector-flows-netobserv-io-v1alpha1"]
+= FlowCollector [flows.netobserv.io/v1alpha1]
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+Description::
++
+--
+FlowCollector is the Schema for the flowcollectors API, which pilots and configures netflow collection.
+--
+
+Type::
+  `object`
+
+
+
+== Specification
```

Comment:
```suggestion
= Specification
```

Reply:
is it something needed for downstream integration?

Reply:
Sort-of. Aidan was trying to help me figure out why my build was breaking yesterday, and basically we were wondering about header leveling precedence parsing through asciibinder build errors and warnings. In the downstream docs, I removed `= FlowCollector [flows.netobserv.io/v1alpha1]` and instead have `= FlowCollector API specifications`, so that way there's no subheading `Specifications` at level 2 (==). So now its a non issue.

Reply:
Ok - I've updated this PR with some automation so that we shouldn't have to do all these manual edits next time.